client/src/components/Setup/index.tsx

```diff
@@ -114,7 +114,7 @@ export default class Setup extends React.Component<any, ISetupState> {
       protocol = 'http';
     }
 
-    return protocol + '//' + host + '/auth/openid/return';
+    return protocol + '://' + host + '/auth/openid/return';
```

Comment:
You're right!
Can you fix the https ==> https: instead? This way it would make better sense.